### PR TITLE
Russian Update

### PR DESCRIPTION
**Pull request by Danik2343**
Russian Update

### CODE REVIEW

**Comment by Croden1999** [comment]
oh my gosh it is.

**Comment by Zangetsu38** [comment]
Changé format kill comparaison

File: lang/ru.xml [hunk 1]
```diff
@@ -1,366 +1,399 @@
...
+		<line_seventh_part_one>Проконсультиция с коммерческой игрой</line_seventh_part_one>
+		<compatibility>совместимость</compatibility>
+		<line_seventh_part_two>Список совместимости и список совместимости с Homebrew, чтобы увидеть, совместимость.</line_seventh_part_two>
+		<line_eighth>Взносы приветствуются!</line_eighth>
+		<line_tenth>Дополнительную поддержку можно найти на канале #help</line_tenth>
+		<line_eleventh>Vita3K не потворствует пиратству. Вы должны дампить свои собственные игры.</line_eleventh>
+		<show_next_time>Показать в следующий раз</show_next_time>
+		<close>Закрыть</close>
+	</welcome>
+</lang>
```
Comment (Zangetsu38) [inline lang/ru.xml:399]: neede new line here

**Comment by Zangetsu38** [comment]
@Danik2343 you can fix formating and force push ?

**Review by Croden1999 — CHANGES_REQUESTED** [review]
there is wrong format.

**Comment by Croden1999** [comment]
plz push again, sir.

**Comment by Croden1999** [comment]
should close this pr, is outdated

**Pull request closed without merging**